packages/features/bookings/components/event-meta/Members.tsx

```diff
@@ -30,23 +30,26 @@ export const EventMembers = ({ schedulingType, users, profile, entity }: EventMe
     !users.length ||
     (profile.name !== users[0].name && schedulingType === SchedulingType.COLLECTIVE);
 
+  const orgAvatarItem =
+    entity.orgSlug && !(isDynamic && !profile.image)
+      ? [
+          {
+            // We don't want booker to be able to see the list of other users or teams inside the embed
+            href: isEmbed ? null : getBookerBaseUrlSync(entity.orgSlug),
+            image: profile.image || "",
+            alt: profile.name || "",
+            title: profile.name || "",
+          },
+        ]
+      : [];
+
   return (
     <>
       <AvatarGroup
         size="sm"
         className="border-muted"
         items={[
-          ...(isDynamic && !profile.image
-            ? []
-            : [
-                {
-                  // We don't want booker to be able to see the list of other users or teams inside the embed
-                  href: isEmbed ? null : getBookerBaseUrlSync(entity.orgSlug),
-                  image: profile.image || "",
-                  alt: profile.name || "",
-                  title: profile.name || "",
-                },
-              ]),
+          ...orgAvatarItem,
           ...shownUsers.map((user) => ({
```

Comment:
`shownUsers` is empty array for a non-org user event. That should also be fixed. We should show the user avatar for such an event and that should take to the user profile page.

This is what the reporter also mentioned here.
![image](https://github.com/calcom/cal.com/assets/1780212/a5fd2bd4-d6fe-42b4-8ff4-106bc6af4755)